pkg/uploader/kopia/progress.go

```diff
@@ -77,11 +79,17 @@ func (p *KopiaProgress) UploadedBytes(numBytes int64) {
 func (p *KopiaProgress) Error(path string, err error, isIgnored bool) {
 	if isIgnored {
 		atomic.AddInt32(&p.ignoredErrorCount, 1)
+		p.errs = append(p.errs, fmt.Errorf("Ignored error when processing %v: %v", path, err))
```

Comment:
Since `p.errs` is just for log printing in line `131@func (kp *kopiaProvider) RunBackup`, why don't we print the log in place here?
If we print the log at the end of the backup, the meaning of the time for each log will be eliminated. 
Instead, if we print the log in place, from the time of the log, we know when the error happens, which is even useful for a long running backup.

Reply:
Meanwhile, p.ignoredErrorCount and p.fatalErrorCount looks a good candidate to print by the end of the backup, e.g., in line `131@func (kp *kopiaProvider) RunBackup`

Reply:
it would be not suitable to output logs in basic public functions and structures, but we could add a timestamp to the error.

Reply:
In fact, I debug with the Kopia client, and through the error output I found the related [Error function ](https://github.com/kopia/kopia/blob/v0.10.4/cli/cli_progress.go#L90).

And our Error in KopiaProgress is imitated from it.

Reply:
> it would be not suitable to output logs in basic public functions and structures

Didn't quite get this, but as the built-in handler [Error function ](https://github.com/kopia/kopia/blob/v0.10.4/cli/cli_progress.go#L90), it is also printing the log directly in the progress.Error callback.